src/main/java/com/fcgl/madrid/forum/service/PostService.java

```diff
@@ -117,12 +146,23 @@ public PostService(PostRepository postRepository) {
     /**
      *
      * @param ex Exception
-     * @return ResponseEntity<InternalStatus>
+     * @return ResponseEntity<GetCityPostsResponse>
      */
     private ResponseEntity<GetCityPostsResponse> fallback(GetCityPostsRequest request, Exception ex) {
         String message = "Fallback: " + ex.getMessage();
         InternalStatus internalStatus = new InternalStatus(StatusCode.UNKNOWN, 500, message);
         GetCityPostsResponse postResponse = new GetCityPostsResponse(internalStatus, null);
         return new ResponseEntity<GetCityPostsResponse>(postResponse, HttpStatus.INTERNAL_SERVER_ERROR);
     }
+
+    /**
+     *
+     * @param ex Exception
+     * @return ResponseEntity<GetCityPostsResponse>
+     */
+    private ResponseEntity<InternalStatus> fallback(PostLikeRequest request, Exception ex) {
```

Comment:
We need to change these from regular Exceptions because we could see a bad spike. We should also log these everytime